app/controllers/people_controller.rb

```diff
@@ -55,6 +55,12 @@ def new
     @person = Person.new(status: cookies[:status], state: 'MA')
     @person.emails.build(category: 'E-Mail', status: 'OK', usage: '1-All')
     @person.phones.build(category: "Mobile Phone", status: "OK", usage: "1-All")
+
+    departments = Department.active.managing_people
+    if @person.department.nil? && departments.count == 1
```

Comment:
Do we need the @person.department.nil? test ? How could it have a value at this point, as we just created it.